src/constants.ts

```diff
@@ -15,13 +15,16 @@ export enum runtimes {
     dotnetcore = 'dotnetcore',
     ruby = 'ruby',
     tomcat = 'tomcat',
-    javase = 'java|8-jre8'
+    javase = 'java|8-jre8',
+    python = 'python'
 }
 
 export function getIgnoredFoldersForDeployment(runtime: string): string[] {
     switch (runtime) {
         case runtimes.node:
             return ['node_modules{,/**}'];
+        case runtimes.python:
+            return ['.env', '.Python', 'env/{,/**}', 'build{,/**}', 'dist{,/**}'];
```

Comment:
Why do some folders have `{,/**}` and some don't?

Reply:
`.env` is one of the most common names for python virtual environments. It's definitely a folder

Reply:
I thought `.env` was a file describing environment variables https://www.ibm.com/support/knowledgecenter/en/ssw_aix_72/com.ibm.aix.osdevice/env_file.htm where env is the folder for virtual environments